homework7/Calculator/Calculator/CalculatorForm.Designer.cs

```diff
@@ -0,0 +1,527 @@
+            button0.FlatStyle = FlatStyle.Flat;
+            button0.Font = new Font("Segoe UI Semilight", 15.75F, FontStyle.Regular, GraphicsUnit.Point);
+            button0.Location = new Point(72, 203);
+            button0.Name = "button0";
+            button0.Size = new Size(63, 47);
+            button0.TabIndex = 18;
+            button0.Text = "0";
+            button0.UseVisualStyleBackColor = false;
+            button0.Click += NumberOrOperationButton_Click;
+            // 
+            // changeSignButton
+            // 
+            changeSignButton.BackColor = SystemColors.ControlLightLight;
+            changeSignButton.Cursor = Cursors.Hand;
+            changeSignButton.Dock = DockStyle.Fill;
+            changeSignButton.FlatAppearance.BorderSize = 0;
+            changeSignButton.FlatStyle = FlatStyle.Flat;
+            changeSignButton.Font = new Font("Segoe UI Semilight", 15.75F, FontStyle.Regular, GraphicsUnit.Point);
+            changeSignButton.Location = new Point(3, 203);
+            changeSignButton.Name = "changeSignButton";
+            changeSignButton.Size = new Size(63, 47);
+            changeSignButton.TabIndex = 19;
+            changeSignButton.Text = "+/-";
+            changeSignButton.UseVisualStyleBackColor = false;
+            changeSignButton.Click += ChangeSignButton_Click;
+            // 
+            // CalculatorForm
+            // 
+            AutoScaleDimensions = new SizeF(7F, 15F);
+            AutoScaleMode = AutoScaleMode.Font;
+            ClientSize = new Size(284, 361);
+            Controls.Add(tableLayoutPanel1);
+            Icon = (Icon)resources.GetObject("$this.Icon");
+            MaximizeBox = false;
+            MaximumSize = new Size(1000, 700);
+            MinimumSize = new Size(300, 400);
+            Name = "CalculatorForm";
+            Text = "Calculator";
+            tableLayoutPanel1.ResumeLayout(false);
+            tableLayoutPanel2.ResumeLayout(false);
+            tableLayoutPanel2.PerformLayout();
+            tableLayoutPanel3.ResumeLayout(false);
+            ResumeLayout(false);
+        }
+
+        #endregion
+
+        private TableLayoutPanel tableLayoutPanel1;
+        private TableLayoutPanel tableLayoutPanel2;
+        private TextBox expressionOnTopTextBox;
+        private TextBox displayTextBox;
+        private TableLayoutPanel tableLayoutPanel3;
+        private Button button1;
+        private Button button2;
+        private Button button3;
+        private Button button4;
+        private Button button5;
+        private Button button6;
+        private Button button7;
+        private Button button8;
+        private Button button9;
+        private Button clearButton;
+        private Button squareButton;
+        private Button reciprocalButton;
+        private Button divideButton;
+        private Button MultiplyButton;
+        private Button SubtractButton;
+        private Button AddButton;
```

Comment:
```suggestion
        private Button multiplyButton;
        private Button dubtractButton;
        private Button addButton;
```
Единообразие в нейминге